handbook/security.md

```diff
@@ -0,0 +1,256 @@
+
+#### Miscellaneous security settings
+
+| #     | Setting                                                      |
+| ----- | ------------------------------------------------------------ |
+| 2.5.6 | Ensure limit ad tracking is enabled                          |
+| 2.10  | Ensure secure keyboard entry terminal.app is enabled         |
+| 5.1.4 | Ensure library validation is enabled                         |
+| 6.3   | Ensure automatic opening of safe files in Safari is disabled |
+
+**Why?**
+
+* Limiting ad tracking has privacy benefits, and no downside.
+* Protecting keyboard entry into the terminal app could prevent malicious applications, or non-malicious but inappropriate applications from receiving passwords.
+* Library validation ensures that applications can't be tricked into loading a library in a different location, leaving it open to abuse.
+* Safari opening files automatically can lead to negative scenarios where files are downloaded and automatically opened in another application. Though the setting relates to files deemed "safe", it includes PDFs and other file formats where malicious documents exploiting vulnerabilities have been seen before.
+
+**User experience impact**
+
+* Minimal to invisible for these settings, however, applications used to create custom keyboard macros will not be able to receive keystrokes when Terminal.app is the active application window.
+
+
+#### Enforce DNS-over-HTTPs (DoH)
+
+| #  | Setting                |
+| -- | ---------------------- |
+| NA | Enforce [DNS over HTTPS](https://en.wikipedia.org/wiki/DNS_over_HTTPS) |
+
+**Why?**
+
+* We assume laptops are used on dangerous networks. Therefore, DNS queries could be exposed, and leak private data. An attacker on the same wireless network could see DNS queries, determine who your employer is, or even intercept them and [respond with malicious answers](https://github.com/iphelix/dnschef). Using DoH protects the DNS queries from eavesdropping and tampering.
+* We use Cloudflare's DoH servers with basic malware blocking. No censorship should be applied on these servers, except towards destinations known as malware related.
+
+
+**User experience impact**
+
+* Misconfigured captive portals, such as in hotels, could be unreachable, as they perform an attack on DNS traffic to redirect you, and misuse IP addresses such as *1.1.1.1*. You can work around this by performing a *nslookup* in the terminal for any domain and manually browsing to the IP being provided as the response. **This should be rare, and getting rarer by the day. The best workaround is to use tethering on a phone or hotspot**.
+* Rare false positives could happen, preventing access to a site. Please report those, as if they become frequent, we will define a strategy for handling them.
+* If your ISP's DNS service goes down, you'll be able to continue working 😎
```

Comment:
I like the emoji. Feel free to include them in other places. They make me feel safe 😄